cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/spark/SparkCqlEvaluatorArgs.java

```diff
@@ -95,4 +95,7 @@
     
     @Parameter(names = { "--key-parameters" }, description = "One or more parameter names that should be included in the parameters column for output rows that are generated.", required = false)
     public List<String> keyParameterNames = null;
+
+    @Parameter(names = { "--correlation-id" }, description = "Identifier consistent between all logs of a single job", required = false)
```

Comment:
I would tweak this description to be a little more clear to users who are not already familiar with what this does. Maybe something like:  "This correlation ID will be written with any log messages created by the application and also to the batch summary file that is created as output." -> Obviously the batch summary part would be dependent on if we add that, per Matt's comment.

Reply:
Changed!